src/spritecache.cpp

```diff
@@ -205,7 +205,10 @@ uint GetSpriteCountForFile(const std::string &filename, SpriteID begin, SpriteID
 	for (SpriteID i = begin; i != end; i++) {
 		if (SpriteExists(i)) {
 			SpriteCache *sc = GetSpriteCache(i);
-			if (sc->file == file) count++;
+			if (sc->file == file) {
+				count++;
+				Debug(sprite, 4, "Sprite: {}", i);
```

Comment:
Feels like this message could be more descriptive. "Fallback sprite in use: " perhaps?

Reply:
Might be, but i decided against it because this function seems to be written for a more generic purpose, even though i could only find this one case where it's being used

Reply:
Yeah, that's the only place it's used. I'd rename & move it, tbh.

At the very least, you could add a corresponding Debug "header" message where it's called

Reply:
Ok, did that.